recipes/prov/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "1.4.0" %}
+
+package:
+  name: prov
+  version: {{ version }}
+
+source:
+  fn: prov-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/1d/bb/9fd5313a33d55fb02a5f885407d6025b7d6fa1ad1b65713107756edba4db/prov-{{ version }}.tar.gz
```

Comment:
So, PyPI changed the URLs on us some time ago. It makes them a little harder to keep up-to-date though. Try the one below.

```
https://pypi.io/packages/source/p/prov/prov-1.4.0.tar.gz
```